src/catalog/src/durable/objects/state_update.rs

```diff
@@ -277,19 +278,19 @@ impl StateUpdateKind {
 pub struct StateUpdateKindJson(Jsonb);
 
 impl StateUpdateKindJson {
-    pub(crate) fn from_serde<S: serde::Serialize>(s: &S) -> Self {
+    pub(crate) fn from_serde<S: serde::Serialize>(s: S) -> Self {
         let serde_value = serde_json::to_value(s).expect("valid json");
         let row =
             Jsonb::from_serde_json(serde_value).expect("contained integers should fit in f64");
         StateUpdateKindJson(row)
     }
 
-    pub(crate) fn to_serde<D: serde::de::DeserializeOwned>(&self) -> D {
+    pub(crate) fn to_serde<D: serde::de::DeserializeOwned>(self) -> D {
         self.try_to_serde().expect("jsonb should roundtrip")
     }
 
     pub(crate) fn try_to_serde<D: serde::de::DeserializeOwned>(
-        &self,
+        self,
     ) -> Result<D, serde_json::error::Error> {
         let serde_value = self.0.as_ref().to_serde_json();
         serde_json::from_value::<D>(serde_value)
```

Comment:
It's very possible that these `serde` methods end up cloning the values anyway, it just seemed silly to pass an `&_` when all of the callers had an owned value.